app/containers/LanguageProvider/tests/index.test.js

```diff
@@ -48,4 +45,33 @@ describe('<LanguageProvider />', () => {
       queryByText(messages.someMessage.defaultMessage),
     ).toBeInTheDocument();
   });
+
+  it.each(Object.keys(translationMessages))(
+    'should render the specified "%s" language messages',
+    locale => {
+      const languageState = { locale };
+      const mockedState = {
+        language: languageState,
+      };
+      const { queryByText } = render(
+        <Provider store={configureStore(mockedState)}>
+          <LanguageProvider messages={translationMessages}>
+            <FormattedMessage {...messages.someMessage} />
+          </LanguageProvider>
+        </Provider>,
+      );
+      if (
+        locale === 'en' ||
+        translationMessages[locale][messages.someMessage.id]
+      ) {
```

Comment:
This test assumes that `en` will always have a test "translation" different from `defaultMessage`, but won't fail if any languages added in the future don't have one defined. That being said, if another language does have a translation defined, but the rendered value is different than expected for the locale, it should still throw an error.